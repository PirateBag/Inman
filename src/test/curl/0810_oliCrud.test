
===JSON===

{ "rows" : [
        {  "activityState" : "INSERT",  "itemId" : "1", "parentOliId" : "0", "quantityOrdered" : "500", "quantityAssigned" : "0.0",
        "completeDate" : "2025-0915",  "orderState" : "OPEN",
        "orderType" : "MOHEAD" }
] }
