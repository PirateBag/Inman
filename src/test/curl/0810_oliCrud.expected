{
  "responseType" : "MULTILINE",
  "errors" : [ {
    "key" : 1,
    "message" : "Added 8 lines to order 67"
  } ],
  "data" : [ {
    "id" : 67,
    "crudAction" : "INSERT",
    "itemId" : 1,
    "quantityOrdered" : 500.0,
    "quantityAssigned" : 0.0,
    "startDate" : "2025-0913",
    "completeDate" : "2025-0915",
    "parentOliId" : 0,
    "orderState" : "OPEN",
    "orderType" : "MOHEAD"
  } ]
}